src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/citizen/PaidInFullTest.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmcts.cmc.claimstore.tests.functional.citizen;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.cmc.claimstore.idam.models.User;
+import uk.gov.hmcts.cmc.claimstore.tests.BaseTest;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.PaidInFull;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PaidInFullTest extends BaseTest {
```

Comment:
How about adding a test case when paid in full is attempted on claim/case 
1. which doesn't exist in database?
2. When user is not a owner of the case?

Reply:
These scenarios are covered in unit tests. Added functional tests as well.